### PR TITLE
Alpha/card-professionals

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Updated media-box component
* Added new modifiers to media-box
* Added new elements to media-box
* Updated demos

screenshot capture - 2017-04-28 - 16-07-30

screenshot capture - 2017-04-28 - 16-07-45

screenshot capture - 2017-04-28 - 16-07-52

screenshot capture - 2017-04-28 - 16-08-00

### CODE REVIEW

**Comment by iondrimba** [comment]
[(5) Update da versão 2 (design) do Gaiden](https://trello.com/c/k2izpoRz/406-5-update-da-versao-2-design-do-gaiden)

File: docs/demo/gaiden-css/components/card/centered.html [hunk 1]
```diff
@@ -13,21 +13,20 @@
...
+    <div class="media-box__title">
+      <h3 class="caption-title">
+        Marcos
+        <small>Boa Viagem - Recife</small>
+      </h3>
+    </div>
   </header>
-  <div class="card__content">
-    <p>- Apartamento com pintura nova branca (imóvel com 2 anos). Repintura do forro de gesso e paredes. Forro de gesso da sala com partes sem massa corrida (sanca embutida)</p>
+  <div class="card__content space-element-large">
```
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/components/card/centered.html]: @iondrimba Is `` `div``` needed, does it not apply as class in` `` p``?

File: docs/demo/gaiden-css/components/card/custom.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    body {
+      background-color: #eff4fa;
+    }
+
+  </style>
+</head>
+
+<body>
+  <div class="container">
+    <div class="col-small-6">
```
Comment (MarcoBrunoBR) [inline docs/demo/gaiden-css/components/card/custom.html]: @iondrimba this class `` `col-small-6``` can not be placed in` `` article```?

File: src/scss/components/_icon.scss [hunk 3]
```diff
@@ -1,10 +1,14 @@
 .icon {
-  display: inline-block;
-  margin: 2px 20px 0;
+  display: flex;
   vertical-align: top;
```
Comment (MarcoBrunoBR) [inline src/scss/components/_icon.scss]: @iondrimba do we need this ```vertical-align```?

Comment (iondrimba) [inline src/scss/components/_icon.scss]: @MarcoBrunoBR no we don't need it. I fixed this on de alpha/card-update PR :)

File: docs/demo/gaiden-css/components/card/centered.html (hunk 1 above)
Comment (iondrimba) [inline docs/demo/gaiden-css/components/card/centered.html]: @MarcoBrunoBR Great tip! You are right, we don't need the extra div. Updated all demos without it :1st_place_medal:

File: docs/demo/gaiden-css/components/card/custom.html (hunk 2 above)
Comment (iondrimba) [inline docs/demo/gaiden-css/components/card/custom.html]: @MarcoBrunoBR Nice tip! You are right, we don't need the extra wrapper. I will change this only for this demo. Maybe another scenario could arise where we could need one. 👍

**Comment by iondrimba** [comment]
@MarcoBrunoBR Any other changes needed?

File: src/scss/components/_card.scss [hunk 4]
```diff
@@ -26,10 +26,25 @@
 .card {
   background-color: get-color(air);
   border-radius: $borderRadius;
-  box-shadow: 0 4px 8px 0 get-color((color: blackNeutral, opacity: .12));
+  box-shadow: 0 2px 4px 0 get-color((color: blackNeutral, opacity: .12));
```
Comment (iondrimba) [inline src/scss/components/_card.scss]: @eduardojmatos on the merge I will keep only one file 👍

**Pull request merged**